src/error.rs

```diff
@@ -83,9 +84,9 @@ impl std::fmt::Display for Error {
             Error::MissingSetupPy {} => {
                 "setup.py not found.\n You may want to run `dmenv init` now".to_string()
             }
-            Error::MissingLock {} => format!(
+            Error::MissingLock { expected_path } => format!(
                 "{} not found.\n You may want to run `dmenv lock` now",
-                LOCK_FILE_NAME
+                expected_path.to_string_lossy()
```

Comment:
Nit: Since the purpose is to format and print it to the user, I might `.display()` instead of `.to_string_lossy()`